transport-native-io_uring/src/test/java/io/netty/channel/uring/IOUringSocketHalfClosedTest.java

```diff
@@ -27,4 +31,12 @@
     protected List<TestsuitePermutation.BootstrapComboFactory<ServerBootstrap, Bootstrap>> newFactories() {
         return IOUringSocketTestPermutation.INSTANCE.socket();
     }
+
+    @Ignore
+    @Test
+    public void testAutoCloseFalseDoesShutdownOutput() throws Throwable {
```

Comment:
And this one?

Reply:
I ignored this one for now as it makes some assumption about execution that is not really valid and more an implementation detail.